src/data/markdown/docs/01 guides/01 Getting started/04 Results Output.md

```diff
@@ -1,77 +1,122 @@
 
 ```shell
-http_req_duration.....: avg=46.32ms, max=46.32ms, med=46.32ms, min=46.32ms, p90=46.32ms, p95=46.32ms
+data_received..............: 148 MB 2.5 MB/s
+data_sent..................: 1.0 MB 17 kB/s
+http_req_blocked...........: avg=1.92ms   min=1µs      med=5µs      max=288.73ms p(90)=11µs     p(95)=17µs
+http_req_connecting........: avg=1.01ms   min=0s       med=0s       max=166.44ms p(90)=0s       p(95)=0s
+http_req_duration..........: avg=143.14ms min=112.87ms med=136.03ms max=1.18s    p(90)=164.2ms  p(95)=177.75ms
+http_req_receiving.........: avg=5.53ms   min=49µs     med=2.11ms   max=1.01s    p(90)=9.25ms   p(95)=11.8ms
+http_req_sending...........: avg=30.01µs  min=7µs      med=24µs     max=1.89ms   p(90)=48µs     p(95)=63µs
+http_req_tls_handshaking...: avg=0s       min=0s       med=0s       max=0s       p(90)=0s       p(95)=0s
+http_req_waiting...........: avg=137.57ms min=111.44ms med=132.59ms max=589.4ms  p(90)=159.95ms p(95)=169.41ms
+http_reqs..................: 13491  224.848869/s
+iteration_duration.........: avg=445.48ms min=413.05ms med=436.36ms max=1.48s    p(90)=464.94ms p(95)=479.66ms
+iterations.................: 13410  223.498876/s
+vus........................: 100    min=100 max=100
+vus_max....................: 100    min=100 max=100
 ```
 
 </div>
 
-After the HTTP timing metrics, there will be a few final lines of output:
+> To learn more about the metrics k6 collects and reports, read the [Metrics guide](/using-k6/metrics).
 
-- `http_reqs........: 2` The total number of HTTP requests made during the whole load test.
-- `iterations........: 1` The total number of times all VUs in the test managed to run through the default() function.
-- `vus.................: 1` How many VUs the test was configured to simulate.
-- `vus_max........: 1` The number of pre-allocated VU slots the test was configured for (vus_max allows you to scale up the number of VUs in the test to max that number).
+**Output of trend metrics**
 
-## Output result data
+[Trend metrics](/using-k6/metrics#metric-types) collect trend statistics (min/max/avg/percentiles) for a series of values. On stdout they are printed like this:
 
-k6 may also output more granular result data using special output plugins. Currently, there are a few plugins that can output data:
+<div class="code-group" data-props='{"labels": []}'>
+
+```shell
+http_req_duration..........: avg=143.14ms min=112.87ms med=136.03ms max=1.18s    p(90)=164.2ms  p(95)=177.75ms
+```
+
+</div>
+
+You could use the [summary-trend-stats](/using-k6/options#summary-trend-stats) option to change the stats reported for Trend metrics.
+
+
+<div class="code-group" data-props='{"labels": []}'>
+
+```shell
+k6 run --summary-trend-stats="avg,p(99)" script.js
+```
+
+</div>
+
+
+
+## Output plugins
+
+k6 can send more granular result data to different outputs to integrate and visualize k6 metrics on other platforms.
+
+The list of output plugins are:
+
+| | |
```

Comment:
I suggest using the headings "Plugin" and "Usage". Without any headings looks a bit broken and is not very accessibility friendly.